update_unix.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"io/ioutil"
+	"net/http"
+	"os"
+	"os/exec"
+	"path/filepath"
+)
+
+// RunUpdate runs the update on the current tgf executable
+func RunUpdate() bool {
+	executablePath, err := os.Executable()
+	if err != nil {
+		printWarning("Error getting executable path", err)
+	}
+
+	currentDir := filepath.Dir(executablePath)
+
+	os.Setenv("TGF_PATH", currentDir)
+
+	updateScript, err := fetchUpdateScript()
+	if err != nil {
+		printWarning("Error fetching update script: ", err)
+	}
+
+	output, err := exec.Command("bash", "-c", updateScript).Output()
```

Comment:
Use `CombinedOutput`. It was failing on my end and I didn't know why. You are missing the stderr with `Output`

Reply:
Oh 
Okay